includes/definitions/discovery/gaia.yaml

```diff
@@ -65,7 +66,7 @@ modules:
                     states:
                         - { value: 1, descr: active , graph: 3, generic: 0 }
                         - { value: 2, descr: standby, graph: 2, generic: 0 }
-                        - { value: 3, descr: 'active attention', graph: 1, generic: 1 }
+                        - { value: 3, descr: 'Active Attention', graph: 1, generic: 1 }
```

Comment:
I was testing and maybe I found a issue here, I think the discovery does not treat case sensitive

-bash-4.2$ snmpwalk -v2c -c public 10.10.10.20 -m /opt/librenms/mibs/checkpoint/CHECKPOINT-MIB haState
CHECKPOINT-MIB::haState.0 = STRING: Active Attention
-bash-4.2$

Before:
![image](https://user-images.githubusercontent.com/79482451/117227366-0c4a9180-aded-11eb-86d0-68ae303731e4.png)

After I've changed to capital a it worked:
![image](https://user-images.githubusercontent.com/79482451/117227390-18365380-aded-11eb-902a-853bdd11055e.png)